# Resolution: Transfer Response Model

Consequent Reversibility was not included in the final escape models because, in this subgroup structure, it is partially encoded by Goal_Type and produced unstable estimates when entered together with Goal_Type.

Import Libraries

In [2]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

## Overall Subgroup
This subset of data explores observations when _Focus_ = I and _Focus_= They.
1. Create resolved_transfer
2. Counts and Proportions
3. Logistic Models
    - RT1: Agency
    - RT2: Goal Type
    - RT3: Additive Model
    - RT4: Interactive Model 
4. Model Comparison
5. Summary Table

Read Data

In [3]:
subgroup_theoretical = pd.read_csv("../../data/processed/subgroup_theoretical.csv")

1. Create resolved_transfer

Binary Outcome: include L1_transfer responses from escape_L2 observations.

In [4]:
subgroup_theoretical["escape_L2"] = (
    subgroup_theoretical["Response_Full"] != "L2_other"
).astype(int)

In [5]:
escapees = subgroup_theoretical[subgroup_theoretical["escape_L2"] == 1].copy()

In [6]:
escapees["resolved_transfer"] = (escapees["Response_Full"] == "L1_transfer").astype(int)

2. Counts and Proportions

In [7]:
resolved_transfer_counts = pd.crosstab(
    escapees["resolved_transfer"],
    escapees["Response_Full"])

resolved_transfer_counts

Response_Full,L1_transfer,correct,missing_response
resolved_transfer,,,
0,0,61,3
1,96,0,0


In [8]:
resolved_transfer_props = escapees["resolved_transfer"].value_counts(normalize = True)
resolved_transfer_props

resolved_transfer
1    0.6
0    0.4
Name: proportion, dtype: float64

From 160 observations, 60% of responses that had escaped *L2_other* options were resolved by restoring to L1_transfer. 

Counts and Proportions by Condition:

In [9]:
counts_condition = pd.crosstab(
    [escapees["Goal_Type"], escapees["Agent"]],
    escapees["resolved_transfer"],
    margins = True)

counts_condition

resolved_transfer         0   1  All
Goal_Type         Agent             
goal_frequent     0      15  13   28
                  1      10  12   22
goal_non_frequent 0      12  21   33
                  1       7  16   23
no_goal           0      11  23   34
                  1       9  11   20
All                      64  96  160

In [10]:
props_condition = pd.crosstab(
    [escapees["Goal_Type"], escapees["Agent"]],
    escapees["resolved_transfer"],
    normalize= "index")

props_condition

resolved_transfer               0         1
Goal_Type         Agent                    
goal_frequent     0      0.535714  0.464286
                  1      0.454545  0.545455
goal_non_frequent 0      0.363636  0.636364
                  1      0.304348  0.695652
no_goal           0      0.323529  0.676471
                  1      0.450000  0.550000

COMMENT GENERAL PROPORTIONS

3. Logistic Models

Import Libraries

In [11]:
import statsmodels.formula.api as smf
from scipy.stats import chi2

In [12]:
escapees["resolved_transfer"].value_counts()

resolved_transfer
1    96
0    64
Name: count, dtype: int64

PRUEBA 1: focus

In [13]:
P1 = smf.logit(
    "resolved_transfer ~ Focus",
    data=escapees
    ).fit()

print(P1.summary())

Optimization terminated successfully.
         Current function value: 0.665764
         Iterations 4
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                  160
Model:                          Logit   Df Residuals:                      158
Method:                           MLE   Df Model:                            1
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:                 0.01077
Time:                        13:47:47   Log-Likelihood:                -106.52
converged:                       True   LL-Null:                       -107.68
Covariance Type:            nonrobust   LLR p-value:                    0.1278
                    coef    std err          z      P>|z|      [0.025      0.975]
---------------------------------------------------------------------------------
Intercept         0.2091      0.205      1.019      0.308      -0.193       0.611
Focus[T.They]     0.

PRUEBA 2: Agent can predict nothing. Focus + Goal Type

In [46]:
P2 = smf.logit(
    "resolved_transfer ~ Focus + Goal_Type",
    data=escapees
    ).fit()

print(P2.summary())

Optimization terminated successfully.
         Current function value: 0.656088
         Iterations 5
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                  160
Model:                          Logit   Df Residuals:                      156
Method:                           MLE   Df Model:                            3
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:                 0.02515
Time:                        15:10:10   Log-Likelihood:                -104.97
converged:                       True   LL-Null:                       -107.68
Covariance Type:            nonrobust   LLR p-value:                    0.1438
                                     coef    std err          z      P>|z|      [0.025      0.975]
--------------------------------------------------------------------------------------------------
Intercept                         -0.2042      0.315     -0.647     

In [48]:
P3 = smf.logit(
    "resolved_transfer ~ Focus * Agent + Goal_Type",
    data=escapees
    ).fit()

print(P3.summary())

Optimization terminated successfully.
         Current function value: 0.643309
         Iterations 5
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                  160
Model:                          Logit   Df Residuals:                      154
Method:                           MLE   Df Model:                            5
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:                 0.04413
Time:                        15:10:46   Log-Likelihood:                -102.93
converged:                       True   LL-Null:                       -107.68
Covariance Type:            nonrobust   LLR p-value:                   0.09055
                                     coef    std err          z      P>|z|      [0.025      0.975]
--------------------------------------------------------------------------------------------------
Intercept                         -0.4614      0.374     -1.232     

**Model RT1**

resolved_transfer ~ Agent

In [14]:
RT1 = smf.logit(
    "resolved_transfer ~ Agent",
    data=escapees
    ).fit()

print(RT1.summary())

Optimization terminated successfully.
         Current function value: 0.673012
         Iterations 4
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                  160
Model:                          Logit   Df Residuals:                      158
Method:                           MLE   Df Model:                            1
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:               1.359e-10
Time:                        13:47:47   Log-Likelihood:                -107.68
converged:                       True   LL-Null:                       -107.68
Covariance Type:            nonrobust   LLR p-value:                    0.9999
                 coef    std err          z      P>|z|      [0.025      0.975]
------------------------------------------------------------------------------
Intercept      0.4055      0.209      1.936      0.053      -0.005       0.816
Agent      -9.339e-16      0.

**Model RT2**

resolved_transfer ~ Goal_Type

In [15]:
RT2 = smf.logit(
    "resolved_transfer ~ Goal_Type",
    data= escapees
    ).fit()

print(RT2.summary())

Optimization terminated successfully.
         Current function value: 0.663269
         Iterations 4
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                  160
Model:                          Logit   Df Residuals:                      157
Method:                           MLE   Df Model:                            2
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:                 0.01448
Time:                        13:47:47   Log-Likelihood:                -106.12
converged:                       True   LL-Null:                       -107.68
Covariance Type:            nonrobust   LLR p-value:                    0.2104
                                     coef    std err          z      P>|z|      [0.025      0.975]
--------------------------------------------------------------------------------------------------
Intercept                      -1.852e-16      0.283  -6.55e-16     

**Model RT3**

resolved_transfer ~ Agent + Goal_Type

In [16]:
RT3 = smf.logit(
    "resolved_transfer ~ Agent + Goal_Type",
    data= escapees
    ).fit()

print(RT3.summary())

Optimization terminated successfully.
         Current function value: 0.663253
         Iterations 4
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                  160
Model:                          Logit   Df Residuals:                      156
Method:                           MLE   Df Model:                            3
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:                 0.01450
Time:                        13:47:47   Log-Likelihood:                -106.12
converged:                       True   LL-Null:                       -107.68
Covariance Type:            nonrobust   LLR p-value:                    0.3731
                                     coef    std err          z      P>|z|      [0.025      0.975]
--------------------------------------------------------------------------------------------------
Intercept                         -0.0103      0.318     -0.032     

**Model RT4**

resolved_transfer ~ Agent * Goal_Type

In [17]:
RT4 = smf.logit(
    "resolved_transfer ~ Agent * Goal_Type",
    data=escapees
    ).fit()

print(RT4.summary())

Optimization terminated successfully.
         Current function value: 0.658907
         Iterations 5
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                  160
Model:                          Logit   Df Residuals:                      154
Method:                           MLE   Df Model:                            5
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:                 0.02096
Time:                        13:47:47   Log-Likelihood:                -105.43
converged:                       True   LL-Null:                       -107.68
Covariance Type:            nonrobust   LLR p-value:                    0.4781
                                           coef    std err          z      P>|z|      [0.025      0.975]
--------------------------------------------------------------------------------------------------------
Intercept                               -0.1431      0.3

4. Model Comparisons

In [18]:
from scipy.stats import chi2

def lr_test(model_small, model_large):

    LR = 2 * (model_large.llf - model_small.llf)

    df = model_large.df_model - model_small.df_model

    p = chi2.sf(LR, df)

    return pd.Series({
        "LR": LR,
        "df": df,
        "p_value": p
    })

In [19]:
lr_test(RT1, RT3)

LR         3.122680
df         2.000000
p_value    0.209855
dtype: float64

In [20]:
lr_test(RT1, RT4)

LR         4.513476
df         4.000000
p_value    0.340953
dtype: float64

5. Summary Table

In [21]:
goal_behaviour = ["Combined", "Combined", "Combined", "Combined", "Combined", "Combined", "Combined", "Combined", "Combined", "Combined", "Combined"]

model_name = ["RT1", "RT2", "RT2", "RT3", "RT3", "RT3", "RT4", "RT4", "RT4", "RT4", "RT4"]

outcome = ["resolved_transfer","resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer"]

subgroup = ["Overall", "Overall", "Overall", "Overall", "Overall", "Overall", "Overall", "Overall", "Overall", "Overall", "Overall"]

formula = ["resolved_transfer ~ Agent", "resolved_transfer ~ Goal_Type", "resolved_transfer ~ Goal_Type", "resolved_transfer ~ Agent + Goal_Type", "resolved_transfer ~ Agent + Goal_Type", "resolved_transfer ~ Agent + Goal_Type", "resolved_transfer ~ Agent * Goal_Type", "resolved_transfer ~ Agent * Goal_Type", "resolved_transfer ~ Agent * Goal_Type", "resolved_transfer ~ Agent * Goal_Type", "resolved_transfer ~ Agent * Goal_Type"]

predictor= ["Agent", "goal_non_frequent", "no_goal", "goal_non_frequent", "no_goal", "Agent",  "goal_non_frequent", "no_goal", "Agent", "Agent:goal_non_frequent", "Agent:no_goal"]

coef = [-9.339e-16, 0.6665, 0.5306, 0.6672, 0.5323, 0.0233, 0.7027, 0.8807, 0.3254, -0.0584, -0.8624]

std_error = [0.329, 0.400, 0.399, 0.400, 0.400, 0.332, 0.524, 0.527, 0.572, 0.814, 0.814]

coef_p = [1.000, 0.095, 0.184, 0.095, 0.183, 0.944, 0.180, 0.095, 0.569, 0.943, 0.290]

LLR_p = [0.999, 0.2104, 0.2104, 0.3731, 0.3731, 0.3731, 0.4781, 0.4781, 0.4781, 0.4781, 0.4781]

observations = [160, 160, 160, 160, 160, 160, 160, 160, 160, 160, 160]

Takeaway = []

In [22]:
transfer_summary_overall = pd.DataFrame({
    "Goal_Behaviour": goal_behaviour,
    "Model": model_name,
    "Outcome": outcome,
    "Subgroup": subgroup,
    "Formula": formula, 
    "Predictor": predictor,
    "Coef": coef,
    "Std_err": std_error,
    "Coef_p": coef_p,
    "Model_LLR_p": LLR_p,
    "N": observations
})

transfer_summary_overall

,Goal_Behaviour,Model,Outcome,Subgroup,Formula,Predictor,Coef,Std_err,Coef_p,Model_LLR_p,N
0,Combined,RT1,resolved_transfer,Overall,resolved_transfer ~ Agent,Agent,-9.339000e-16,0.329,1.000,0.9990,160
1,Combined,RT2,resolved_transfer,Overall,resolved_transfer ~ Goal_Type,goal_non_frequent,6.665000e-01,0.400,0.095,0.2104,160
2,Combined,RT2,resolved_transfer,Overall,resolved_transfer ~ Goal_Type,no_goal,5.306000e-01,0.399,0.184,0.2104,160
3,Combined,RT3,resolved_transfer,Overall,resolved_transfer ~ Agent + Goal_Type,goal_non_frequent,6.672000e-01,0.400,0.095,0.3731,160
4,Combined,RT3,resolved_transfer,Overall,resolved_transfer ~ Agent + Goal_Type,no_goal,5.323000e-01,0.400,0.183,0.3731,160
5,Combined,RT3,resolved_transfer,Overall,resolved_transfer ~ Agent + Goal_Type,Agent,2.330000e-02,0.332,0.944,0.3731,160
6,Combined,RT4,resolved_transfer,Overall,resolved_transfer ~ Agent * Goal_Type,goal_non_frequent,7.027000e-01,0.524,0.180,0.4781,160
7,Combined,RT4,resolved_transfer,Overall,resolved_transfer ~ Agent * Goal_Type,no_goal,8.807000e-01,0.527,0.095,0.4781,160
8,Combined,RT4,resolved_transfer,Overall,resolved_transfer ~ Agent * Goal_Type,Agent,3.254000e-01,0.572,0.569,0.4781,160
9,Combined,RT4,resolved_transfer,Overall,resolved_transfer ~ Agent * Goal_Type,Agent:goal_non_frequent,-5.840000e-02,0.814,0.943,0.4781,160


## Actors Subgroup
This subset of data explores observations only when _Focus_ = I.
1. Create resolved_transfer
2. Counts and Proportions
3. Logistic Models
    - RT1_A: Agency
    - RT2_A: Goal Type
    - RT3_A: Additive Model
    - RT4_A: Interactive Model 
4. Model Comparison
5. Summary Table

Read Data

In [23]:
subgroup_actors = subgroup_theoretical[subgroup_theoretical["Focus"] == "I"].copy()

1. Create resolved_transfer

Binary Outcome: include transfer responses from escape_L2 observations within actors.

In [24]:
subgroup_actors["escape_L2"] = (
    subgroup_actors["Response_Full"] != "L2_other"
).astype(int)

In [25]:
escapees_actors = subgroup_actors[subgroup_actors["escape_L2"] == 1].copy()

In [26]:
escapees_actors["resolved_transfer"] = (escapees_actors["Response_Full"] == "L1_transfer").astype(int)

2. Counts and Proportions

In [27]:
resolved_transfer_counts = pd.crosstab(
    escapees_actors["resolved_transfer"],
    escapees_actors["Response_Full"])

resolved_transfer_counts

Response_Full,L1_transfer,correct,missing_response
resolved_transfer,,,
0,0,40,3
1,53,0,0


In [28]:
resolved_transfer_props = escapees_actors["resolved_transfer"].value_counts(normalize = True)
resolved_transfer_props

resolved_transfer
1    0.552083
0    0.447917
Name: proportion, dtype: float64

From 96 actors' observations (from the original 160), 55.2% of responses that managed to escape *L2_other* options were resolved by restoring to *L1_transfer*.

Counts and Proportions by Condition:

In [29]:
counts_condition = pd.crosstab(
    [escapees_actors["Goal_Type"], escapees_actors["Agent"]],
    escapees_actors["resolved_transfer"],
    margins = True)

counts_condition

resolved_transfer         0   1  All
Goal_Type         Agent             
goal_frequent     0      10   6   16
                  1       7   7   14
goal_non_frequent 0       9   7   16
                  1       4  11   15
no_goal           0       8  13   21
                  1       5   9   14
All                      43  53   96

In [30]:
props_condition = pd.crosstab(
    [escapees_actors["Goal_Type"], escapees_actors["Agent"]],
    escapees_actors["resolved_transfer"],
    normalize= "index")

props_condition

resolved_transfer               0         1
Goal_Type         Agent                    
goal_frequent     0      0.625000  0.375000
                  1      0.500000  0.500000
goal_non_frequent 0      0.562500  0.437500
                  1      0.266667  0.733333
no_goal           0      0.380952  0.619048
                  1      0.357143  0.642857

ADD COMMENT ON GENERAL PROPORTIONS 

3. Logistic Models

In [31]:
escapees_actors["resolved_transfer"].value_counts()

resolved_transfer
1    53
0    43
Name: count, dtype: int64

**RT1_A Model** 

resolved_transfer ~ Agent

In [32]:
RT1_A = smf.logit(
    "resolved_transfer ~ Agent",
    data=escapees_actors
    ).fit()

print(RT1_A.summary())

Optimization terminated successfully.
         Current function value: 0.678229
         Iterations 4
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                   96
Model:                          Logit   Df Residuals:                       94
Method:                           MLE   Df Model:                            1
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:                 0.01379
Time:                        13:47:47   Log-Likelihood:                -65.110
converged:                       True   LL-Null:                       -66.020
Covariance Type:            nonrobust   LLR p-value:                    0.1772
                 coef    std err          z      P>|z|      [0.025      0.975]
------------------------------------------------------------------------------
Intercept     -0.0377      0.275     -0.137      0.891      -0.576       0.501
Agent          0.5610      0.

**RT2_A Model** 

resolved_transfer ~  Goal_Type

In [33]:
RT2_A = smf.logit(
    "resolved_transfer ~ Goal_Type",
    data=escapees_actors
    ).fit()

print(RT2_A.summary())

Optimization terminated successfully.
         Current function value: 0.673952
         Iterations 4
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                   96
Model:                          Logit   Df Residuals:                       93
Method:                           MLE   Df Model:                            2
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:                 0.02001
Time:                        13:47:47   Log-Likelihood:                -64.699
converged:                       True   LL-Null:                       -66.020
Covariance Type:            nonrobust   LLR p-value:                    0.2669
                                     coef    std err          z      P>|z|      [0.025      0.975]
--------------------------------------------------------------------------------------------------
Intercept                         -0.2683      0.368     -0.728     

**RT3_A Model**

resolved_transfer ~  Agent + Goal_Type

In [34]:
RT3_A = smf.logit(
    "resolved_transfer ~ Agent + Goal_Type",
    data=escapees_actors
    ).fit()

print(RT3_A.summary())

Optimization terminated successfully.
         Current function value: 0.663120
         Iterations 4
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                   96
Model:                          Logit   Df Residuals:                       92
Method:                           MLE   Df Model:                            3
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:                 0.03576
Time:                        13:47:47   Log-Likelihood:                -63.659
converged:                       True   LL-Null:                       -66.020
Covariance Type:            nonrobust   LLR p-value:                    0.1933
                                     coef    std err          z      P>|z|      [0.025      0.975]
--------------------------------------------------------------------------------------------------
Intercept                         -0.5595      0.427     -1.311     

**RT4_A Model**

resolved_transfer ~ Agent * Goal_Type

In [35]:
RT4_A = smf.logit(
    "resolved_transfer ~ Agent * Goal_Type",
    data= escapees_actors
    ).fit()

print(RT4_A.summary())

Optimization terminated successfully.
         Current function value: 0.656589
         Iterations 5
                           Logit Regression Results                           
Dep. Variable:      resolved_transfer   No. Observations:                   96
Model:                          Logit   Df Residuals:                       90
Method:                           MLE   Df Model:                            5
Date:                Wed, 15 Jul 2026   Pseudo R-squ.:                 0.04526
Time:                        13:47:47   Log-Likelihood:                -63.033
converged:                       True   LL-Null:                       -66.020
Covariance Type:            nonrobust   LLR p-value:                    0.3086
                                           coef    std err          z      P>|z|      [0.025      0.975]
--------------------------------------------------------------------------------------------------------
Intercept                               -0.5108      0.5

4. Model comparisons

In [36]:
lr_test(RT1_A, RT3_A)

LR         2.900943
df         2.000000
p_value    0.234460
dtype: float64

In [37]:
lr_test(RT1_A, RT4_A)

LR         4.154880
df         4.000000
p_value    0.385451
dtype: float64

5. Summary Table

transfer_summary_actors

In [38]:
goal_behaviour = ["Combined", "Combined", "Combined", "Combined", "Combined", "Combined", "Combined", "Combined", "Combined", "Combined", "Combined"]

model_name = ["RT1_A", "RT2_A", "RT2_A", "RT3_A", "RT3_A", "RT3_A", "RT4_A", "RT4_A", "RT4_A", "RT4_A", "RT4_A"]

outcome = ["resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer", "resolved_transfer"]

subgroup = ["Actors", "Actors", "Actors", "Actors", "Actors", "Actors", "Actors", "Actors", "Actors", "Actors", "Actors"]

formula = ["resolved_transfer ~ Agent", "resolved_transfer ~ Goal_Type", "resolved_transfer ~ Goal_Type", "resolved_transfer ~ Agent + Goal_Type", "resolved_transfer ~ Agent + Goal_Type", "resolved_transfer ~ Agent + Goal_Type", "resolved_transfer ~ Agent * Goal_Type", "resolved_transfer ~ Agent * Goal_Type", "resolved_transfer ~ Agent * Goal_Type", "resolved_transfer ~ Agent * Goal_Type", "resolved_transfer ~ Agent * Goal_Type"]

predictor= ["Agent", "goal_non_frequent", "no_goal", "goal_non_frequent", "no_goal", "Agent",  "goal_non_frequent", "no_goal", "Agent", "Agent:goal_non_frequent", "Agent:no_goal"]

coef = [0.5610, 0.5937, 0.7944, 0.5968, 0.8534, 0.6113, 0.2595, 0.9963, 0.5108, 0.7521, -0.4085]

std_error =[0.418, 0.518, 0.508, 0.524, 0.516, 0.427, 0.722, 0.685, 0.743, 1.071, 1.032]

coef_p = [0.180, 0.252, 0.118, 0.255, 0.098, 0.153, 0.719, 0.146, 0.492, 0.483, 0.692]

LLR_p = [0.1772, 0.2669, 0.2669, 0.1933, 0.1933, 0.1933, 0.3086, 0.3086, 0.3086, 0.3086, 0.3086]

observations = [96, 96, 96, 96, 96, 96, 96, 96, 96, 96, 96]

Takeaway = []

In [39]:
transfer_summary_actors = pd.DataFrame({
    "Goal_Behaviour": goal_behaviour,
    "Model": model_name,
    "Outcome": outcome,
    "Subgroup": subgroup,
    "Formula": formula, 
    "Predictor": predictor,
    "Coef": coef,
    "Std_err": std_error,
    "Coef_p": coef_p,
    "Model_LLR_p": LLR_p,
    "N": observations,
})

transfer_summary_actors

,Goal_Behaviour,Model,Outcome,Subgroup,Formula,Predictor,Coef,Std_err,Coef_p,Model_LLR_p,N
0,Combined,RT1_A,resolved_transfer,Actors,resolved_transfer ~ Agent,Agent,0.5610,0.418,0.180,0.1772,96
1,Combined,RT2_A,resolved_transfer,Actors,resolved_transfer ~ Goal_Type,goal_non_frequent,0.5937,0.518,0.252,0.2669,96
2,Combined,RT2_A,resolved_transfer,Actors,resolved_transfer ~ Goal_Type,no_goal,0.7944,0.508,0.118,0.2669,96
3,Combined,RT3_A,resolved_transfer,Actors,resolved_transfer ~ Agent + Goal_Type,goal_non_frequent,0.5968,0.524,0.255,0.1933,96
4,Combined,RT3_A,resolved_transfer,Actors,resolved_transfer ~ Agent + Goal_Type,no_goal,0.8534,0.516,0.098,0.1933,96
5,Combined,RT3_A,resolved_transfer,Actors,resolved_transfer ~ Agent + Goal_Type,Agent,0.6113,0.427,0.153,0.1933,96
6,Combined,RT4_A,resolved_transfer,Actors,resolved_transfer ~ Agent * Goal_Type,goal_non_frequent,0.2595,0.722,0.719,0.3086,96
7,Combined,RT4_A,resolved_transfer,Actors,resolved_transfer ~ Agent * Goal_Type,no_goal,0.9963,0.685,0.146,0.3086,96
8,Combined,RT4_A,resolved_transfer,Actors,resolved_transfer ~ Agent * Goal_Type,Agent,0.5108,0.743,0.492,0.3086,96
9,Combined,RT4_A,resolved_transfer,Actors,resolved_transfer ~ Agent * Goal_Type,Agent:goal_non_frequent,0.7521,1.071,0.483,0.3086,96
